AsyncDisplayKit/ASTextNode.mm

```diff
@@ -749,6 +749,33 @@ - (UIImage *)placeholderImage
 
 #pragma mark - Touch Handling
 
+-(BOOL)pointInside:(CGPoint)point withEvent:(UIEvent *)event
+{
+  if (!_passthroughNonlinkTouches) {
+    return [super pointInside:point withEvent:event];
+  }
+
+  NSRange range = NSMakeRange(0, 0);
+  NSString *linkAttributeName = nil;
+  BOOL inAdditionalTruncationMessage = NO;
+
+  id linkAttributeValue = [self _linkAttributeValueAtPoint:point
+                                             attributeName:&linkAttributeName
+                                                     range:&range
+                             inAdditionalTruncationMessage:&inAdditionalTruncationMessage];
+
+  NSUInteger lastCharIndex = NSIntegerMax;
```

Comment:
Hmm, this doesn't seem right, does it?  Not only does the logic look odd using this value, why use the local variable at all if it is just NSIntegerMax?